sentry-android-core/src/main/java/io/sentry/android/core/UserInteractionIntegration.java

```diff
@@ -82,16 +89,23 @@ private void stopTracking(final @Nullable Window window) {
   public void onActivityCreated(@NotNull Activity activity, @Nullable Bundle bundle) {}
 
   @Override
-  public void onActivityStarted(@NotNull Activity activity) {}
+  public void onActivityStarted(@NotNull Activity activity) {
+    // The docs on the screen rendering performance tracing
+    // (https://firebase.google.com/docs/perf-mon/screen-traces?platform=android#definition),
+    // state that the tracing starts for every Activity class when the app calls .onActivityStarted.
+    // Adding an Activity in onActivityCreated leads to Window.FEATURE_NO_TITLE not
+    // working. Moving this to onActivityStarted fixes the problem.
+    activityFramesTracker.addActivity(activity);
+  }
 
   @Override
   public void onActivityResumed(@NotNull Activity activity) {
-    startTracking(activity.getWindow(), activity);
+    startTracking(activity);
   }
 
   @Override
   public void onActivityPaused(@NotNull Activity activity) {
-    stopTracking(activity.getWindow());
+    stopTracking(activity);
```

Comment:
Shouldn't it be when the transaction is finished? `ActivityLifecycleIntegration` only does on destroyed, iirc.

Reply:
do you mean when the activity is finished? Nope, because the WindowCallback/GestureListener only makes sense when the activity is interact-able, otherwise we remove it in onPause and re-attach onResume, so it makes sense to finish the UI transaction here as well, as the activity is not interact-able anyway?

Reply:
Yeah makes sense, my bad.